addons/l10n_ua_ifrs/data/account_chart_template.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <record id="ua_ias_transfer_account" model="account.account.template">
+        <field name="name">Транзитний рахунок</field>
+        <field name="code">1119</field>
+        <field eval="True" name="reconcile"/>
+        <field name="user_type_id"
+            ref="account.data_account_type_current_assets"/>
+    </record>
+
+    <!-- Chart template -->
+    <record id="l10n_ua_ias_chart_template" model="account.chart.template">
+        <field name="name">План рахунків МСФЗ</field>
+        <field name="bank_account_code_prefix">1112</field>
+        <field name="cash_account_code_prefix">1111</field>
```

Comment:
Actually there is no need anymore to create a transfer account like you did, you can just specify in the account.chart.template model a field with name "transfer_account_code_prefix" and the code used for that account
example: <field name="transfer_account_code_prefix">1119</field>

Reply:
Okay, I will improve that